bridge/jni/src/errors.rs

```diff
@@ -0,0 +1,117 @@
+use dsnp_graph_config::errors::DsnpGraphError;
+use jni::{objects::JThrowable, JNIEnv};
+use std::fmt;
+
+#[derive(Debug)]
+pub enum SdkJniError {
+	DsnpGraph(DsnpGraphError),
```

Comment:
Not sure if in JNI we should also return the `error_codes` we have set

Reply:
Great idea